engine/src/main/java/com/datatorrent/stram/plan/logical/LogicalPlan.java

```diff
@@ -405,6 +412,7 @@ public void sendMetrics(Collection<String> metricNames)
     private String persistOperatorName;
     public Map<InputPortMeta, OperatorMeta> sinkSpecificPersistOperatorMap;
     public Map<InputPortMeta, InputPortMeta> sinkSpecificPersistInputPortMap;
+    private String moduleName;  // Name of the module which has this stream. null if top level stream.
```

Comment:
This module name is used in REST api to get the list of streams populated by the module, while returning module DAG in REST api.